### ISSUE TITLE
AttributeError for the logger

### ISSUE THREAD

**Issue by martinRenou** [QuantStack/xeus-python#24]
I had this error:
```
AttributeError: 
'xeus_python_logger.XPythonLogger' object has no attribute 'flush'
```
I don't know how to reproduce it. Maybe we should implement `flush`, if it makes sense.

**Comment by SylvainCorlay** [comment]
Yes I think that it makes sense to implement `flush`. You could also try out not sending the message until flush is called.

**Comment by martinRenou** [comment]
According to the [documentation](https://docs.python.org/2/library/sys.html#sys.stderr):

>  stdout and stderr needn’t be built-in file objects: any object is acceptable as long as it has a write() method that takes a string argument.

Anyway, I think you're right and we should implement a flush method, because it's there in the original `stdout` and in `ipython`'s.

**Comment by martinRenou** [comment]
I tried locally. I implemented a `flush` method, and I kept a buffer containing the messages while sending them only when `flush` is called. And apparently flush is not called. At least not when using the `print` function.

**Comment by martinRenou** [comment]
This is fixed #89
